### DISCUSSION TITLE
Trust Self-Signed Certificate?

### DISCUSSION THREAD

**Discussion by chimeforest** [Murmele/Gittyup#371] [Q&A]
Hello,

I'm hosting my own private Gitea instance on my own TrueNAS server in my house. I'm using traefik and pihole to help route stuff.
The TrueNAS server has a Self-Signed Certificate, but Gittyup won't accept it when cloning a repository. Is there a way to force Gittyup to accept it?

Alternatively, I've gotten a Cloudflare origins certificate for it. My browsers accept that certificate, is there a way to get Gittyup to accept it too? With this certificate, Gittyup won't even let me save my Gitea account credentials("SSL handshake failed. The issuer certificate of a locally looked up certificate could not be found. No certificates could be verified.") let alone clone a repository("invalid certificate").

Any help is appreciated, 
Thank you =]

**Comment by exactly-one-kas** [comment]
Gittyup should use the OS's certificate store to validate certificates, you can try storing your self-signed certificate there. Which OS are you using? 

I'd advise against putting Cloudflare's root certificate in the certificate store since anyone can create cloudflare certificates for any domain names

**Reply by chimeforest** [reply]
Thanks for the information!
I'm using Windows 10 and also EndeavourOS  22-9 With KDE Plasma as the desktop.

**Reply by Murmele** [reply]
@chimeforest do you have the problems on both systems? On Linux it might be because of the flatpak package

**Reply by chimeforest** [reply]
@Murmele Yes, I have the problem on both systems. Sorry for the long delay, I got really frustrated and gave up. I tried exactly-one-kas's suggestion, but it didn't work for me.
